index.html

```diff
@@ -1,140 +1,170 @@
+	<!-- Bootstrap CSS-->
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.3/css/bootstrap.min.css">
+	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700">
+	<!-- Custom Stylesheet -->
+	<link rel="shortcut icon" href="favicon.ico">
+	<link rel="stylesheet" href="css/custom.css">
+	<!--[if lt IE 9]>
 		<script src="js/html5shiv.js"></script>
 		<script src="js/respond.js"></script>
 		<![endif]-->
-	</head>
-	<body>
-		<header>
-			<nav class="content">
-				<a href="index.html"><h1 class="gittext lowercase">Git<span>for Windows</span></h1></a>
-				<div class="version"><a href="https://github.com/git-for-windows/git/releases/tag/v2.34.0.windows.1" title="Version 2.34.0 was published on Mon, 15 Nov 2021 19:52:38 UTC">Version 2.34.0</a></div>
-				<ul class="list-unstyled">
-					<li><a style="font-weight:bold" href="https://github.com/git-for-windows/git/wiki/FAQ" target="_blank">FAQ</a></li>
-					<li><a href="https://github.com/git-for-windows/git" target="_blank">Repository</a></li>
-					<li><a href="https://groups.google.com/group/git-for-windows" target="_blank">Mailing List</a></li>
-				</ul>
-			</nav>
-			<section class="content feature">
-				<div class="imagerollover">
-					<img src="img/gwindows_logo.png" alt="Git Logo">
-					<img class="top" src="img/git_logo.png" alt="Git Logo">
+  </head>
+
+  <body>
+	<!-- navbar -->
+	<nav class="navbar navbar-light navbar-expand-lg fixed-top shadow-sm bg-white">
+		<a href="index.html" class="navbar-brand">
+			<h1 class="gittext lowercase">
+				Git<span>for Windows</span>
+			</h1>
+		</a>
+		<button type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+			aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation"
+			class="navbar-toggler"><span class="navbar-toggler-icon"></span></button>
+		<div id="navbarSupportedContent" class="collapse navbar-collapse">
+			<ul class="navbar-nav ml-auto">
+				<li class="nav-item"><a style="font-weight:bold" href="https://github.com/git-for-windows/git/wiki/FAQ"
+						target="_blank" class="nav-link">FAQ</a></li>
+				<li class="nav-item"><a href="https://github.com/git-for-windows/git" target="_blank"
+						class="nav-link">Repository</a></li>
+				<li class="nav-item"><a href="https://groups.google.com/group/git-for-windows" target="_blank"
+						class="nav-link">Mailing List</a></li>
+				<li class="nav-item"><a href="https://gitter.im/git-for-windows/git" target="_blank"
+						class="nav-link">Gitter</a></li>
+				<li class="nav-item"><a href="https://groups.google.com/g/git-mentoring" target="_blank"
+						class="nav-link">Git Mentoring</a></li>
+			</ul>
+		</div>
+	</nav>
+	<section class="bg-light">
+		<div class="container">
+			<div class="row">
+				<div class="col-lg-6 order-2 order-lg-1">
+					<img src="img/gwindows_logo.png" alt="Git Logo" class="img-fluid">
```

Comment:
When the logo is large, or even just viewed on a HiDPI screen it looks blurry. We should probably use an SVG here so it's always crisp.